### PR TITLE
Improving the way images work in Docs

### PR DESCRIPTION
**Pull request by JKarlavige**
[Asana task](https://app.asana.com/0/1200099998847559/1202225356712129/f)

## Description & motivation
This overhauls our `Lightbox` image component, adding additional props to improve control of images within our documentation pages.

No changes to existing images are made within this PR. The best way to test is to pull this branch locally and verify the available Lightbox image props are working as expected. The available props are noted in the documentation below, along with a Loom video walking through the changes.

This also includes a new Notion document covering how to work with the Lightbox component, and what options are available. The documentation is available [here](https://www.notion.so/dbtlabs/Lightbox-Component-19d4496c073542868dee7bf33ff59b30).

Loom walkthrough: https://www.loom.com/share/2b64dbd47a2d46dbafa5b43ed52a91e0

## Previews
First two images should still be side-by-side as is on the live site: https://deploy-preview-2508--docs-getdbt-com.netlify.app/docs/collaborate/manage-access/set-up-sso-azure-active-directory

### CODE REVIEW

File: website/src/components/lightbox/index.js [hunk 1]
```diff
@@ -1,36 +1,49 @@
...
+          ${imageAlignment}
+        `}
+        style={width && {maxWidth: width}}
+      >
+        <span>
+          <a href="#" data-featherlight={src}>
+            <img
+              data-toggle="lightbox"
+              alt={alt ? alt : title ? title : ''}
+              title={title ? title : ''}
```
Comment (john-rock) [inline website/src/components/lightbox/index.js:37]: Nice use of ternary operator here.

**Review by john-rock — APPROVED** [review]
All lgtm! Nice work!

**Comment by runleonarun** [comment]
Hey @JKarlavige I will look at this today!

**Comment by JKarlavige** [comment]
> Hey @JKarlavige I will look at this today!

Sounds good thank you!

**Review by runleonarun — APPROVED** [review]
@JKarlavige This looks amazing! Thank you so much for getting these added!!

**Pull request merged**